src/electron/process_manager.ts

```diff
@@ -96,8 +80,7 @@ async function isSsLocalReachable() {
 //  - silently restart tun2socks when the system is about to suspend (Windows only)
```

Comment:
This comment (and the corresponding log message in `suspendListener`) seems to be inaccurate.  Instead, it seems like the code silently restarts tun2socks after the system resumes.  Do we know if this is actually necessary?  It seems like it would be simpler if we could skip that restart (which, together with the UDP check, creates a double-restart).

Reply:
Per my comment above, we now only (re)start tun2socks once (no extra start for UDP). Unfortunately, it is necessary to restart tun2socks on resume because the process terminates on suspend due to the TAP device getting closed. 

I have updated the comment to accurately reflect this behavior.